metsim/metsim.py

```diff
@@ -66,12 +67,20 @@
                   'standard_name': 'precipitation_flux'},
          'shortwave': {'units': 'W m-2', 'long_name': 'shortwave radiation',
                        'standard_name': 'surface_downwelling_shortwave_flux'},
-         'lwrad': {'units': 'W m-2', 'long_name': 'longwave radiation',
-                   'standard_name': 'surface_downwelling_longwave_flux'},
+         'longwave': {'units': 'W m-2', 'long_name': 'longwave radiation',
+                      'standard_name': 'surface_downwelling_longwave_flux'},
          't_max': {'units': 'C', 'long_name': 'maximum daily air temperature',
                    'standard_name': 'daily_maximum_air_temperature'},
          't_min': {'units': 'C', 'long_name': 'minimum daily air temperature',
                    'standard_name': 'daily_minimum_air_temperature'},
+         'temp': {'units': 'C', 'long_name': 'air temperature',
+                  'standard_name': 'air_temperature'},
+         'vapor_pressure': {'units': 'mbar', 'long_name': 'vapor pressure',
+                            'standard_name': 'vapor_pressure'},
+         'tskc': {'units': 'fraction', 'long_name': 'cloud fraction',
+                  'standard_name': 'cloud_fraction'},
+         'rel_humid': {'units': '%', 'long_name': 'relative humidity',
+                       'standard_name': 'relative_humidity'},
          '_global': {'conventions': '1.6', 'title': 'Output from MetSim',
                      'institution': 'University of Washington',
                      'source': 'metsim.py',
```

Comment:
I think it probably makes sense to make all of these OrderedDict type for consistency.